src/flows/context/flow.list.tsx

```diff
@@ -307,6 +307,11 @@ export const FlowListProvider: FC = ({children}) => {
   )
 
   const migrate = async () => {
+    const localFlows = Object.keys(flows).filter(id => id.includes('local'))
+    if (!localFlows.length) {
```

Comment:
this was causing a double rerender for all notebooks because it pushes a new state into localstorage right off the bat

Reply:
honestly we could get rid of all this migrate garbage, right? that was there to help migrate notebooks to the API way back in the day when notebooksd was first born

Reply:
idk. it did come in handy when we were doing some migrations right? i think it might all break (if it doesn't already) when @asalem1 starts dropping in the "and you need a websocket channel" stuff, so lets do it then